### ISSUE TITLE
Default branch settings for an organisation

### ISSUE THREAD

**Issue by alexanderadam** [go-gitea/gitea#12040]
## Description

It would be great if it would be possible to add some defaults for organisations. For instance protected branches that require [signed commits](https://github.com/go-gitea/gitea/pull/9708) and disable pushes to the `main` branch.

**Comment by mikeperalta1** [comment]
+1 I currently have to use `DEFAULT_BRANCH` key in app.ini to set the default branch, but that only covers the global case. It would be great if each organization can override the default branch setting.

**Comment by bendem** [comment]
Agreed with this, we want to prevent force push on the default branch in our organisation, this is currently not possible.
